content/blog/authors.json

```
@@ -79,7 +79,7 @@
   {
     "name": "Nudonge",
     "slug": "nudonge",
-    "role": "moderator",
+    "role": "regular",
```

```suggestion
    "role": "verified",
```
CommunityRegularとなる場合、権限はVerifiedになります。